scheme.sql

```diff
@@ -0,0 +1,100 @@
+CREATE DATABASE taskforce CHARACTER SET utf8 COLLATE utf8_general_ci;
+
+USE taskforce;
+
+CREATE TABLE users (
+    id INT AUTO_INCREMENT PRIMARY KEY,
+    email CHAR(64),
+    name CHAR(64),
+    password CHAR(256),
+    dt_add DATETIME,
```

Comment:
```suggestion
    created_at DATETIME,
    updated_at DATETIME,
```
не эконом на буквах. Правило: если есть created_at должен быть и updated_at. Иногда можно пренебречь, если обновлений не планируется.